content/patterns/button/examples/js/button.js

```diff
@@ -24,7 +24,6 @@ function init() {
 
 /**
  * Activates the action button with the enter key.
- *
```

Comment:
I think we sould preserve the current way these JSDoc lines are handled by explicitly setting a [jsdoc/tag-lines](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/tag-lines.md#tag-lines) rule.

Setting `"jsdoc/tag-lines": ["warn", "any", { "startLines":1 }]` in .eslintrc.json rules should be what we want.